components/blocks/CompareBox.tsx

```diff
@@ -0,0 +1,346 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { FaCircle } from 'react-icons/fa'
+import Slider from 'react-slick'
+import 'slick-carousel/slick/slick.css'
+import 'slick-carousel/slick/slick-theme.css'
+import css from 'styled-jsx/css'
```

Comment:
Lets stick to either `StyledComponents` or Tailwind

Reply:
as per our conversation due to targetting specific components of the slick carousel we are keeping the styled jsx